template/codeland.rb

```diff
@@ -122,6 +126,7 @@ def rails_4?
 gem 'slim-rails', '~> 3.0.1'
 run 'bundle install --quiet'
 application %Q(config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }), :env => 'development'
+application  'config.action_mailer.delivery_method = :letter_opener', :env => 'development'
```

Comment:
Line is too long. [91/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.